UM/View/RenderBatch.py

```diff
@@ -247,7 +247,6 @@ def _renderItem(self, item: Dict[str, Any]):
             normal_matrix = Matrix(transformation.getData())
             normal_matrix.setRow(3, [0, 0, 0, 1])
             normal_matrix.setColumn(3, [0, 0, 0, 1])
-            normal_matrix.invert()
```

Comment:
I would expect that removing an invert has some serious behavioral change.
Was the invert placed there in error?

Reply:
The invert was never needed. As far as I've been able to find from the code, we just care about the general angle here (not the actual direction). This invert was also the most expensive part of the calculations here, so this is probably one of the most influential changes wrt speed (especially when you render a lot of items at once) in this PR.